doc/oper.xml

```diff
@@ -1503,3 +1503,41 @@ true
   </Description>
 </ManSection>
 <#/GAPDoc>
+  
+<#GAPDoc Label="HamiltonianPath">
+<ManSection>
+  <Prop Name="HamiltonianPath" Arg="digraph"/>
+  <Returns>A List or <K>fail</K>.</Returns>
+  <Description>
+    Returns a <E>Hamiltonian</E> path if one exists, <K>fail</K> if not.
+
+    A <E>Hamiltonian</E> path is a directed circuit visiting
+    each vertex exactly once. See Section <Ref Subsect="Definitions"
+     Style="Number" /> for the definition of a directed circuit.
+    Note the empty digraphs on 0 and 1 vertices are considered
+    Hamiltonian.<P/>
+
+    The method used in this operation has worst case complexity <M>O(n!)</M>
```

Comment:
I am not sure that this is correct, since you no longer use the naive DFS approach. Maybe remove this sentence altogether.